wallet-new/src/Cardano/Wallet/Orphans.hs

```diff
@@ -0,0 +1,19 @@
+{- |
+   An orphanage for all the non-interesting orphan instances.
+-}
+{-# OPTIONS_GHC -fno-warn-orphans #-}
+module Cardano.Wallet.Orphans where
```

Comment:
I believe that you can actually name this `Cardano.Wallet.Aeson` and put here all Aeson instances which wallet-new uses. We do so with `Bi` instances in Core, we do so with JSON instances in old wallet and I still think that this is a good approach.

Reply:
I think that's a good suggestion. As long as we have a uniform scheme for dealing with that, I'm fine sticking to well-throttled conventions.